bom/pom.xml

```diff
@@ -166,6 +181,56 @@
          </dependency>
          <dependency>
             <groupId>${project.groupId}</groupId>
+            <artifactId>infinispan-distexec-demo</artifactId>
+            <version>${project.version}</version>
+         </dependency>
+         <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>infinispan-gridfs-webdav</artifactId>
+            <version>${project.version}</version>
```

Comment:
If they want to build the demo, they build it as part of the tree, not import it as a dependency.